index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>caclulator</title>
+  </head>
+  <body>  
+     
+      <header class="header">
+          <div class="header_inner container">                       
+            <a class="logo" href="#">
+                <img src="#" alt="logo">
+            </a> 
+           <div class="input-inner">
+            <span>Найти продукт: </span>
+            <input type="text" id= 'search-product' list="product-list" autocomplete="on">
```

Comment:
id= 'search-product' list="product-list" -> id="search-product" list="product-list"

use only (") type of quotes inside *.html files

Reply:
done